fsharp-backend/src/BackendOnlyStdLib/LibEvent.fs

```diff
@@ -30,18 +29,17 @@ let fns : List<BuiltInFn> =
         (function
         | state, [ data; DStr space; DStr name ] ->
           uply {
-            // See client/src/Entry.ml for the "_"
+            // See client/src/Entry.ml for the "_" WHATISTHIS
```

Comment:
There's a "_", and this file explains why. Or did at some point.

The reason is that handlers in the DB have 3 fields (eg Http, /path, GET), but we don't need a 3rd one for workers.